src/NuGet.Core/NuGet.Commands/Strings.resx

```diff
@@ -765,4 +835,36 @@ For more information, visit http://docs.nuget.org/docs/reference/command-line-re
     <value>The repository with service index '{0}' does not list any trusted certificate.</value>
     <comment>0 - service index</comment>
   </data>
+  <data name="Error_CouldNotAdd" xml:space="preserve">
+    <value>Could not add trusted signer: {0}</value>
+    <comment>0 - Reason</comment>
+  </data>
+  <data name="Error_CouldNotList" xml:space="preserve">
+    <value>Could not list trusted signers: {0}</value>
+    <comment>0 - Reason</comment>
+  </data>
+  <data name="Error_CouldNotRemove" xml:space="preserve">
+    <value>Could not remove trusted signer: {0}</value>
+    <comment>0 - Reason</comment>
+  </data>
+  <data name="Error_CouldNotSync" xml:space="preserve">
+    <value>Could not sync trusted repository: {0}</value>
+    <comment>0 - Reason</comment>
+  </data>
+  <data name="SuccessfullUpdatedTrustedSigner" xml:space="preserve">
+    <value>Successfully updated trusted signer with name: '{0}'.</value>
```

Comment:
There's some inconsistency in the strings that I've noticed. 

Some say with name, others just have '{0}'. 

Should the "with name" parts be removed?